assets/js/google-translate.js

```diff
@@ -27,8 +27,7 @@ const languages = [
   ["vi", "Vietnamese"]
 ];
 
-export default () =>
-  window.addEventListener("load", handleTranslation);
+export default () => window.addEventListener("load", handleTranslation);
```

Comment:
Same as above. Are these changes intnetional?

Reply:
My formatter did it! Seems to pass the suite though

Reply:
Okay, I bring it up because this can muddle the git history as no logic was changed here.  Its a small thing, but I wanted to bring it up